tests/qa_automation/execute_test_run.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
pleasefix copyright to:
`Copyright © 2017 SUSE LLC`

Reply:
Copy